conda/environments/cuml_dev_cuda10.0.yml

```diff
@@ -9,7 +9,7 @@ dependencies:
 - libclang=8.0.0
 - cmake=3.14.5
 - numba=0.45*
-# - cupy>=6*
+- rapidsai/label/cuda10.0::cupy>=6*
```

Comment:
might as well just require >=6.2? we won't test earlier

Reply:
The channel only contains 6.2 and 6.3. Since this is the developer yml can change that in a follow pr to not delay this one.